server/models/blog.js

```diff
@@ -0,0 +1,39 @@
+const { db } = require ('../db/config')
```

Comment:
db shouldn't be in { }, we noticed that when we tried to follow the path /blog/1 and /blog/2 it would return "blog not found" and Dragos spotted this, we took it out of the { } and the path now works!

Reply:
we noticed that just a few moments ago. latest commit fixes that issue :)